public/app/plugins/panel/geomap/layers/data/routeLayer.tsx

```diff
@@ -110,10 +153,10 @@ export const routeLayer: MapLayerRegistryItem<RouteConfig> = {
         radius: crosshairRadius,
         stroke: new Stroke({
           color: alpha(style.base.color, 0.4),
-          width: crosshairRadius + 2
+          width: crosshairRadius + 2,
         }),
-        fill: new Fill({color: style.base.color}),
-      })
+        fill: new Fill({ color: style.base.color }),
```

Comment:
^^ should use the fixed color

(though style.base.color may actually have it!) just commenting without working with the code